libs/fabric/src/lib/components/combo-box/base-combo-box.component.ts

```diff
@@ -20,7 +20,6 @@ import { ComboBoxOptionsDirective } from './directives/combo-box-options.directi
 
 export abstract class FabBaseComboBoxComponent extends ReactWrapperComponent<IComboBoxProps>
   implements OnInit, AfterContentInit {
-  @ContentChild(ComboBoxOptionDirective) readonly optionsDirective?: ComboBoxOptionDirective;
```

Comment:
Why was this removed?

Reply:
I'm the one who put it there in the first place. It doesn't do anything.

Reply:
Ok then 👍.

See the rest of my comments, but looks good. Feel free to merge when you're happy with it. The most important thing is to not break existing users that don't use the declarative API (but instead the imperative `options` one). Did you check that these still work?

Reply:
Yes! Imperative operation still works as before. When both are used simultaneously, though, the declarative one 'wins'. I'll add comments to the directives to indicate this.